cmd/controller/keyregistry_test.go

```diff
@@ -0,0 +1,103 @@
+package main
+
+import (
+	"errors"
+	"testing"
+
+	"k8s.io/client-go/kubernetes/fake"
+)
+
+func writeTestKeylistSecret(client *fake.Clientset) error {
+	privkey, cert, err := generatePrivateKeyAndCert(1024)
+	if err != nil {
+		return err
+	}
+	if err := writeKeyRegistry(client, privkey, cert, "namespace", "listname"); err != nil {
```

Comment:
These `writeKeyRegistry` tests are obsolete and should be removed.

(See also travis failures: https://travis-ci.org/bitnami-labs/sealed-secrets/jobs/521534393)